create_books.sql

```diff
@@ -0,0 +1,10 @@
+CREATE DATABASE books_db;
```

Comment:
Consider adding constraints like 'NOT NULL' for fields that should not be empty and adding indexes on frequently queried columns, such as 'title' and 'author', to improve data integrity and query performance.